src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/DataSourceGroupCollectionTests.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using System.Drawing;
+
+namespace System.Windows.Forms.Design.Tests;
+
+    public class DataSourceGroupCollectionTests
+    {
+        private DataSourceGroupCollection CreateCollection() => new();
+        private MockDataSourceGroup CreateMockGroup() => new();
+
+        [Fact]
+        public void Add_ShouldAddItem()
+        {
+            var collection = CreateCollection();
+            var group = CreateMockGroup();
```

Comment:
Could we update to use explicit type here instead of var
```suggestion
            DataSourceGroupCollection collection = CreateCollection();
            MockDataSourceGroup group = CreateMockGroup();
```